### PR TITLE
Issue #66, relabel LTI to eClass.

### LINKED ISSUE #66: Relabel LTI URL tab and text
### Descriptive summary

PROD - when a logged in user views share options, there is an LTI URL tab and explanatory text that is not easily understandable by non-technical users 

### Expected behavior
-LTI URL tab should be relabelled to "eClass"
-Text in this tab should read: "Copy the link below to share this resource in eClass or other learning management systems"

### Steps to reproduce the behavior

1. As a logged in user, go to any record view
1. Click share and go to LTI URL tab

### PR DESCRIPTION
**Pull request by cwant**
Fixes #66 

Relabeling 'LTI' to 'eClass' as specified in Issue #66. Before/after screenshot below.

issue_66

### CODE REVIEW

File: app/views/media_objects/_lti_url.html.erb [hunk 1]
```diff
@@ -1,14 +1,14 @@
 <% if section=='tab-content' %>
 
     <div class="tab-pane" id="lti-tab">
-      <p class="muted">Copy the link below to share this resource via LTI</p>
+      <p class="muted"><%= t('share.lti.copy_link') %></p>
       <form role="form">
         <div class="form-group">
-          <label for="ltilink-object">Item</label>
+          <label for="ltilink-object"><%= t('share.item') %></label>
```
Comment (mbarnett) [inline app/views/media_objects/_lti_url.html.erb]: What's the logic between a couple of these being in an `lti` namespace and a couple not?

Comment (cwant) [inline app/views/media_objects/_lti_url.html.erb]: The 'Item' and 'Section' labels also appear in the 'Share this resource' tab and aren't specific to LTI/eClass. The other labels have text that is LTI specific.

Comment (cwant) [inline app/views/media_objects/_lti_url.html.erb]: (although I didn't actually change the 'Share this resource' tab to use these new translation labels)

**Comment by cwant** [comment]
@mbarnett Ping ...

**Pull request merged**